examples/63_Conic.html

```diff
@@ -36,12 +37,45 @@
 		              {name:"Co", type:"ConicBy5", args:["A","B","C","D","E"],color:[0,0,1],alpha:1,size:1},
 //		     {name:"l", type:"Join", args:["A", "D"]}
                       ];
-            createCindy({canvasname:"CSCanvas",
+            var cdy=createCindy({canvasname:"CSCanvas",
                         movescript:"csmove",
                         grid:1,
                         snap:true,
                         geometry:gslp});
 
+            var cases = [
+                ["parabola1", [0,0,1], [1,1,1], [2,4,1], [-2,4,1], [-1,1,1]],
```

Comment:
I don't see `parabola1` getting drawin in a single stroke. At what level did you instrumemt this? Inside the CindyJS code, or in the Browser somehow? In any case, the termination criterion is checking the area of the convex hull of the two endpoints and the control point. That's likely overestimating the approximation error, particularly in case of a parabola. But I can think of no better error measure which could be computed in reasonable time. The distance between the center point and the conic was something I tried before, and that failed to detect lack of fidelity in almost-degenerate hyperbola situations in particular. Can you think of some other error measure, @elkins0?

The `lineTo` calls I could reproduce. The distance spanned is *very* short, so the tangent lines are almost identical, triggering the `Math.abs(denom) > 1e12` check. I doubt that refinement led to such close points. Instead I assume that due to numeric errors, the two horizontal tangents are seen at slightly different *x* values. We might want to detect such cases, and merge them into one, but on the other hand the current behavior shouldn't have much of a performance impact either, I think.

We might do special handling of circles and circular arcs one day, using cubic approximations with even the endpoints slightly off the curve to reduce overall error. But that's a different matter, to be handled in a separate pull request.